core/templates/dev/head/pages/learner-dashboard-page/learner-dashboard-page.controller.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The Oppia Authors. All Rights Reserved.
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
I think you can leave it as 2017 since we are only renaming the file.
@seanlip I am not sure about this, What do you suggest?

Reply:
It's OK either way, I think.